### ISSUE TITLE
How to configure particular rule?

### ISSUE THREAD

**Issue by zhaparoff** [gajus/eslint-plugin-jsdoc#383]
It is unclear from the README (which is the only documentation source for this plugin, as I assume), how to provide options for a particular rule. For example, how to specify `publicOnly: true` for `required-jsdoc` rule?

Schema which is used for standard ESLint rules doesn't work:
```js
"rule-name": [
  "severity",
  {
    "additionalSetting": false
  }
]
```

For aforementioned `required-jsdoc` rule, if I'm providing config:
```js
    "require-jsdoc": [
      1,
      {
        "publicOnly": true
      }
    ]
```
it gives me error:

> Configuration for rule "require-jsdoc" is invalid:
> Value {"publicOnly":true,"require":{}} should NOT have additional properties.

node: `12.9.0`
npm: `6.11.3`
gulp-eslint: `6.0.0`
ESLint: `6.4.0`
eslint-plugin-jsdoc: `15.9.1`

**Comment by zhaparoff** [comment]
Seems like the issue was in using the deprecated standard ESLint rule name `require-jsdoc` instead of `jsdoc/require-jsdoc`
But I think it would be nice to have an example of rule configuration in the docs. At least a generic one, pattern which is common for all rules.
